interpreter/spec/ast.ml

```diff
@@ -66,6 +75,9 @@ type storeop = Memory.mem_size memop
 type var = int32 Source.phrase
 type literal = Values.value Source.phrase
 
+type loadfieldop = {struct_ : var; field : int32}
```

Comment:
No particular reason to separate load and store version of these. Also, `field` should probably be a `var`, too.